go.mod

```diff
@@ -49,3 +49,5 @@ require (
 	k8s.io/api v0.0.0-20190813020757-36bff7324fb7 // indirect
 	k8s.io/client-go v0.0.0-20181219152756-3dd551c0f083 // indirect
 )
+
+replace github.com/containers/storage => github.com/saschagrunert/storage v1.12.3-0.20191015073819-a34ddea087da
```

Comment:
TODO: Remove this when https://github.com/containers/storage/pull/443 got merged (or not)